pyvista/core/dataset.py

```diff
@@ -2083,7 +2083,7 @@ def find_closest_point(self, point: Iterable[float], n=1) -> int:
             return vtk_id_list_to_array(id_list)
         return locator.FindClosestPoint(point)
 
-    def find_closest_cell(self, point: Union[int, np.ndarray]) -> Union[int, np.ndarray]:
+    def find_closest_cell(self, point: Union[int, np.ndarray], return_closest_point=False) -> Union[int, np.ndarray]:
```

Comment:
Isn't the return type affected by the signature change? We might return a tuple, right?

Reply:
I'm worried that mypy passed with this to begin with, but it is fixed now.  All of the code necessary to appease mypy would be simpler if we force the output to be an `ndarray` or `(ndarray, ndarray)`, but this is a breaking change.